include/odp/api/spec/timer.h

```diff
@@ -173,6 +220,28 @@ typedef struct {
 int odp_timer_capability(odp_timer_clk_src_t clk_src,
 			 odp_timer_capability_t *capa);
 
+/**
+ * Timer resolution capability
+ *
+ * This function fills in capability limits for timer pool resolution and
+ * min/max timeout values, based on either resolution or maximum timeout.
+ * Set the required value to 'res_ns' or 'max_tmo', and set other fields to
+ * zero. A successful call fills in the other two fields. The call returns
+ * a failure, if the user defined value exceeds (max_res or max_tmo) capability
```

Comment:
Move `(max_res or max_tmo)` before exceeds and add missing apostrophes.

Reply:
Fixed